stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/decorator/internal/TextDecorator.kt

```diff
@@ -45,11 +45,6 @@ internal class TextDecorator(val style: MessageListItemStyle) : BaseDecorator()
         viewHolder.binding.apply {
             getStyleTextColor(data.isMine)?.let { textColor ->
                 messageText.setTextColor(textColor)
-                // set title & description colors as well
```

Comment:
It's not right. We have link view in many others VHs, but somewhy we did setup only for `MessagePlainText`. Remove this right now. I will fix setup link attachments text in next PR